src/OrbitBase/include/OrbitBase/ProcessId.h

```diff
@@ -0,0 +1,24 @@
+// Copyright (c) 2020 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef ORBIT_BASE_LINUX_PROC_H_
+#define ORBIT_BASE_LINUX_PROC_H_
```

Comment:
These include guards will have to be fixed (once we settle on the name of the file).

Reply:
done